.github/workflows/eval-ui-tests-p0.yml

```diff
@@ -0,0 +1,104 @@
+name: Maestro UI Tests
```

Comment:
`m`: I would maybe rename this to Critical UI Tests or something, same applies to the workflows.yml, so the naming doesn't reflect the underlying tech we're using but instead the purpose.